sdk/cosmos/azure-cosmos-spark_3_2-12/pom.xml

```diff
@@ -81,10 +81,15 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-serializer-json-jackson</artifactId>
+      <version>1.6.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-serializer-json-jackson;current} -->
```

Comment:
You'll want to use the dependency version of this.

```suggestion
      <version>1.5.0</version> <!-- {x-version-update;com.azure:azure-core-serializer-json-jackson;dependency} -->
```

Reply:
Yeah - Build analyze slapped me on the fingers as well :-) Fixed